commit b1dd3beda9991a02b63c032faefcd4d9331f5f7c
Author: Ilya Tumaykin <itumaykin@gmail.com>
Date:   Sun May 29 20:33:30 2016 +0300

build: don't install tests, only build them

Considering tests are usually executed right after a successful build,
there's no reason to keep them around for later.
---

diff --git a/wscript_build.py b/wscript_build.py
index 3e3ec0b..d4f07fb 100644
--- a/wscript_build.py
+++ b/wscript_build.py
@@ -483,11 +483,12 @@ def build(ctx):
     if ctx.dependency_satisfied('test'):
         for test in ctx.path.ant_glob("test/*.c"):
             ctx(
-                target   = os.path.splitext(test.srcpath())[0],
-                source   = test.srcpath(),
-                use      = ctx.dependencies_use() + ['objects'],
-                includes = _all_includes(ctx),
-                features = "c cprogram",
+                target       = os.path.splitext(test.srcpath())[0],
+                source       = test.srcpath(),
+                use          = ctx.dependencies_use() + ['objects'],
+                includes     = _all_includes(ctx),
+                features     = "c cprogram",
+                install_path = None,
             )

     build_shared = ctx.dependency_satisfied('libmpv-shared')
